### PR TITLE
fix top bar

### LINKED ISSUE #116: Top banner image for sidebar pages is to small
The top banner image should span content area & sidebar for all pages which have a sidebar.

https://main--eder-group--hlxsites.hlx.page/unternehmen/historie
https://main--eder-group--hlxsites.hlx.page/unternehmen/ueber-uns
https://main--eder-group--hlxsites.hlx.page/newstermine/pressemeldungen/detail/baumarkt-prospekt-des-eder-profi-baumarkt-in-tuntenhausen-bad-endorf-und-wasserburg
https://main--eder-group--hlxsites.hlx.page/newstermine/veranstaltungen/detail/auto-prof-gorinchem

### PR DESCRIPTION
**Pull request by hannessolo**
Fix #116

Test URLs:

- https://main--eder-group--hlxsites.hlx.page/unternehmen/historie
- https://main--eder-group--hlxsites.hlx.page/unternehmen/ueber-uns
- https://main--eder-group--hlxsites.hlx.page/newstermine/pressemeldungen/detail/baumarkt-prospekt-des-eder-profi-baumarkt-in-tuntenhausen-bad-endorf-und-wasserburg
- https://main--eder-group--hlxsites.hlx.page/newstermine/veranstaltungen/detail/auto-prof-gorinchem
- https://issue-116--eder-group--hlxsites.hlx.page/unternehmen/historie
- https://issue-116--eder-group--hlxsites.hlx.page/unternehmen/ueber-uns
- https://issue-116--eder-group--hlxsites.hlx.page/newstermine/pressemeldungen/detail/baumarkt-prospekt-des-eder-profi-baumarkt-in-tuntenhausen-bad-endorf-und-wasserburg
- https://issue-116--eder-group--hlxsites.hlx.page/newstermine/veranstaltungen/detail/auto-prof-gorinchem

### CODE REVIEW

**Comment by hannessolo** [comment]
Todo: Verify that linked images still work. @mhaack do you know where there is an example of one?

**Comment by mhaack** [comment]
This one has a linked image in the sidebar: https://issue-116--eder-group--hlxsites.hlx.page/unternehmen/ueber-uns

**Comment by hannessolo** [comment]
> This one has a linked image in the sidebar: https://issue-116--eder-group--hlxsites.hlx.page/unternehmen/ueber-uns

Well, that one is broken anyway... I can fix in while I'm at it.

**Comment by hannessolo** [comment]
@mhaack Before it was broken, because it assumed that there would always be a structure like this:

```
...
p
  picture
p
  a
...
```

But it was also detecting the header because it was just looking for a picture where the parent has another child with an a tag. 

Also, the previous logic missed cases where it was just:

```
p
  picture
  br
  a
```

I think this second case makes more sense and is easier to detect - in the logic I added now, the a tag needs to be in the same paragraph, either directly on the same line as the picture, or separated by a `br` (shift+enter in Word).

**Comment by mhaack** [comment]
I think we should not dictate authors how they place the link below the image. Ideally we can detect both cases. Since this is beyond this PR, if not already fixed lets have a separate PR.

File: scripts/scripts.js [hunk 1]
```diff
@@ -152,15 +158,18 @@ export function convertDate(excelDate) {
...
-  const nextSib = picture.parentNode.nextElementSibling;
-  if (nextSib) {
-    const a = nextSib.querySelector('a');
-    if (a && a.textContent.trim().startsWith('https://')) {
-      a.innerHTML = '';
-      a.className = '';
-      a.appendChild(picture);
-      oldParent.remove();
+  let nextSib = picture.nextElementSibling;
+  console.log(nextSib?.tagName);
```
Comment (mhaack) [inline scripts/scripts.js:162]: Remove log

**Review by mhaack — APPROVED** [review]

**Pull request merged**